docs/frontend/dataviews.md

```
@@ -0,0 +1,796 @@
+# Dokan DataView Table Documentation
+
+- [Introduction](#introduction)
+- [Quick Overview](#quick-overview)
+- [1. Features](#1-role-specifix-controllers)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)